docs/quickstart/services.rst

```diff
@@ -8,16 +8,14 @@ Grafana
 
 You will first need the latest ``kubectl`` version installed on your host.
 
-From the bootstrap node, get the port used by Grafana:
```

Comment:
Indeed, we can remove this section regarding Grafana's port, as it'll always be 3000.